### PR TITLE
Ldap users

### PR DESCRIPTION
**Pull request by KlamChowder1**
view LDAP users stuff 

branched off v-card-header (https://github.com/qurit/raiven/pull/146)

image

### CODE REVIEW

**Comment by KlamChowder1** [comment]
- added access / admin controls to admin page 
admin_and_access
- put file results in own page
image
image

File: backend/api/routes/user.py [hunk 1]
```diff
@@ -14,6 +14,32 @@
...
+def modify_user_access(user_id: int, user: User = Depends(token_auth), db: Session = Depends(session)):
+    """ Change the user's access permissions """
+
+    if not user.is_admin:
+        return HTTPException(403, 'Unauthorized')
+
+    user_to_change = db.query(User).get(user_id)
+    user_to_change.access_allowed = not user_to_change.access_allowed
+    user_to_change.save(db)
+    return user_to_change
```
Comment (awtkns) [inline backend/api/routes/user.py:27]:
```suggestion
@router.put("/modify-access/{user_id}", response_model=UserSchema, dependencies=[Depends(admin_auth)])
def modify_user_access(user_id: int, db: Session = Depends(session)):
    """ Change the user's access permissions """
    
    user_to_change = db.query(User).get(user_id)
    user_to_change.access_allowed = not user_to_change.access_allowed
    user_to_change.save(db)
    return user_to_change
```

Comment (awtkns) [inline backend/api/routes/user.py:27]: You are going to want to check that the user_id exists, else 404

**Review by awtkns — CHANGES_REQUESTED** [review]

**Comment by KlamChowder1** [comment]
image

- added type
- put all edit user settings stuff in one route

**Review by awtkns — APPROVED** [review]
LGTM 😄

**Pull request merged**